src/appengine/handlers/cron/ml_train.py

```diff
@@ -19,6 +19,12 @@
 from datastore import fuzz_target_utils
 from handlers import base_handler
 from libs import handler
+from metrics import logs
+
+MODEL_NAME_TO_TASK = {
+    'gradientfuzz': 'train_gradientfuzz',
```

Comment:
Should we rename gradientfuzz to something like gradient_generator or something similar to rnn_generator ?
Same for train_gradientfuzz files ?

Reply:
I think it's reasonable to leave it as-is, I don't think it's necessary for everything to be prefixed with generator. We made it a point to rename rnn to rnn_generator because it was starting to look like "rnn" was too generic a name to make sense for what we were doing (we can and probably will use RNNs for other things, so we had to be more specific).